### DISCUSSION TITLE
Intel iGPU utilization?

### DISCUSSION THREAD

**Discussion by mkanet** [nicolargo/glances#2309] [Q&A]
I just switched to a new PC that doesn't have an Nvidia video card anymore.  It uses the integrated **Intel UHD770 iGPU** for Plex/Emby hardware acceleration.  How can I monitor GPU load?

EDIT:  I just found the below link.  It looks like there was recent activity just last week!  How close are we to have this feature?
https://github.com/nicolargo/glances/issues/994

**Comment by nicolargo** [comment]
Hi @mkanet 

unfortunately, for the moment Glances did not support Intel GPU.

The feature request is traced in https://github.com/nicolargo/glances/issues/994

Nicolas
